doc/man1/flux-kvs.rst

```diff
@@ -65,170 +65,616 @@ setting the namespace in the environment variable :envvar:`FLUX_KVS_NAMESPACE`.
-
-**wait** [-N ns] *version*
-   Block until the KVS version reaches *version* or greater. A simple form
-   of synchronization between peers is: node A puts a value, commits it,
-   reads version, sends version to node B. Node B waits for version, gets
-   value.
-
-**getroot** [-N ns] [-s \| -o \| -b]
-   Retrieve the current KVS root, displaying it as an RFC 11 dirref object.
-   Specify an alternate namespace to retrieve from via *-N*. If *-o* is
-   specified, display the namespace owner. If *-s* is specified, display
-   the root sequence number.  If *-b* is specified, display the root blobref.
-
-**eventlog get** [-N ns] [-W] [-w] [-c count] [-u] *key*
-   Display the contents of an RFC 18 KVS eventlog referred to by *key*.
-   If *-u* is specified, display the log in raw form. If *-W* is
-   specified and the eventlog does not exist, wait until it has been
-   created. If *-w* is specified, after the existing contents have
-   been displayed, the eventlog is monitored and updates are displayed
-   as they are committed.  This runs until the program is interrupted
-   or an error occurs, unless the number of events is limited with the
-   *-c* option. Specify an alternate namespace to display from via
-   *-N*.
-
-**eventlog append** [-N ns] [-t SECONDS] *key* *name* [*context* ...]
-   Append an event to an RFC 18 KVS eventlog referred to by *key*.
-   The event *name* and optional *context* are specified on the command line.
-   The timestamp may optionally be specified with *-t* as decimal seconds since
-   the UNIX epoch (UTC), otherwise the current wall clock is used.
-   Specify an alternate namespace to append to via *-N*.
-
-**eventlog wait-event** [-N ns] [-t SECONDS] [-u] [-W] [-q] [-v] *key* *event*
-   Wait for a specific *event* to occur in an RFC 18 KVS eventlog
-   referred to by *key*.  If *-t* is specified, timeout after
-   *SECONDS* if the event has not occurred.  If *-u* is specified,
-   display the log in raw form. If *-W* is specified and the eventlog
-   does not exist, wait until it has been created. If *-q* is
-   specified, not output the matched event.  If *-v* is specified,
-   output all events prior to the matched event.  This runs until the
-   program is interrupted, the event occurs, or a timeout occurs if
-   *-t* is specified.  Specify an alternate namespace to display from
-   via *-N*.
+The :program:`flux kvs` sub-commands and their arguments are described below.
+
+get
+---
+
+.. program:: flux kvs get
+
+Retrieve the value stored under *key* and print it on standard output.
+It is an error if *key* does not exist, unless :option:``--waitcreate`` is
+specified.  It is an error if *key* is not a directory.
+
+If multiple *key* arguments are specified, their values are concatenated.
+A newline is appended to each value, unless :option:`--raw` is specified or
+the value is zero length.
+
+
+.. option:: -h, --help
+
+  Display help on this sub-command.
+
+.. option:: -N, --namespace=NAME
```

Comment:
personal preference ... `--help` option feels the least useful in the manpage (i.e. you're already in the manpage).  List it last amongst all the options?

Reply:
Great point. Instead of repeating that option and the namespace one, I'll just mention it in the section intro.